### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: styleguide.css [hunk 1]
```diff
@@ -550,3 +565,24 @@ button.menu-button.active + nav ul li {
...
     display: block;
 }
+
+/* Page profile */
+
+main {
+    display: grid;
+    grid-gap: 2rem;
+    grid-template-columns: 50% 50%;
+    grid-template-areas: 'section-left section-right';
```
Comment (vinyll) [inline styleguide.css]: I always wonder how to name areas.
Do you think names like "left" and "right" names will always be pertinent on every screen size?
This is how [they suggest it on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Grid_Template_Areas).
GridByExamples seems to [follow the same rule](https://gridbyexample.com/examples/).

Another example on [Typanus show an explicit naming example](https://tympanus.net/codrops/css_reference/grid/#section_eg3).

File: styleguide.css [hunk 2]
```diff
@@ -550,3 +565,24 @@ button.menu-button.active + nav ul li {
...
+    border: 0.1rem solid #00283C;
+}
+main section {
+    padding:1.5rem;
```
Comment (vinyll) [inline styleguide.css]: missing a space after `:` ;)

File: styleguide.css (hunk 1 above)
Comment (johangiraud) [reply styleguide.css]: ok, i go for _content_ and _sidebar_

File: styleguide.css (hunk 2 above)
Comment (johangiraud) [reply styleguide.css]: ah oui

**Review by vinyll — APPROVED** [review]

**Pull request merged**